### ISSUE TITLE
CR47 (Exclude 'N/A' from source habitat details)

### ISSUE THREAD

**Issue by AndyFoy** [HabitatFramework/HLUTool#76]
##### Description #####
Currently it is possible to assigned up to three sources to each incid, each of which **must** have one of the available 'Habitat Class' and 'Habitat Type' values selected (to show that these haven't been forgotten).  For situations when a habitat class (e.g. 'NVC') or habitat type (e.g. 'CG2') aren't relevant, for example when the source relates to aerial photography, the user can select the value '**N/A**'.  However, TVERC do not want 'N/A' to appear in any exported datasets.

##### Proposed Solution #####
Adapt the HLU Tool so that values of 'N/A' in the habitat class or habitat type fields for any sources can be replaced with blanks when performing an export.  The option to **hide** 'N/A' values will either be available in the main 'Options' interface or in the 'Export' window (to be determined when developing the solution).

It is also proposed that a new habitat type option of '**Unknown**' is added to the database lookup tables so that, where the habitat class **is known** but the specific habitat type within that class **is not known** then a value of 'Unknown' can be selected for the habitat type rather than needing to select 'N/A'.

**Comment by GrahamHawker** [comment]
When we discussed "Unknown" this was for using as a legacy (Phase 1( category when this wasn't definable based on the available information. 

I don't see that it would be useable or relevant in the circumstances supplied. You would never not know the habitat type because the habitat class is derived from the habitat type. So if someone says improved grassland then it's phase 1 as the class or if they say lowland meadow it's Priority habitats. Unknown wouldn't come up here.

**Comment by bconeybeer** [comment]
This change request is of low priority for HBIC but happy for you to develop it. 
Regarding 'unknown' - I agree with Graham here, I don't really see the need for it.

**Comment by AndyFoy** [comment]
Graham, I obviously misunderstood our conversation.  Are you talking about having 'Unknown' as an option in the 'Legacy' field or in the Sources 'Habitat Type' field?  If the later then does it apply **only** to Phase 1 habitats?  Perhaps you can explain exactly what you need with respect to 'Unknown' habitat types as I don't understand your requirements.

**Comment by GrahamHawker** [comment]
Yes unknown as an option in the legacy field and yes it only applies to phase 1 habitats in the legacy field.

**Comment by drewlawson** [comment]
This would be low priority for SxBRC
